src/sentry/web/urls.py

```diff
@@ -321,6 +322,10 @@ def init_all_applications():
         name='sentry-organization-auth-settings'
     ),
     url(
+        r'^organizations/(?P<organization_slug>[\w_-]+)/integrations/(?P<provider_id>[\w_-]+)/setup/$',
```

Comment:
still not sure about this..

basically the UI needs to POST to an endpoint, which starts the flow. That flow will bind to their current session and then direct them to the next 'step'. Historically that would be something like '/organizations/:slug/integrations/', though given thats rendered by UI we cant overload it. Instead we can either use the API (a bit incorrect, since it would be tied to the session), or setup another endpoint.